package.json

```diff
@@ -26,10 +26,14 @@
     "prettier:fix": "npm run prettier -- --write",
     "format": "npm run prettier:fix && npm run lint:fix",
     "scripts:release-post": "NODE_NO_WARNINGS=1 node scripts/release-post/index.mjs",
-    "test": "cross-env NODE_NO_WARNINGS=1 jest --passWithNoTests",
-    "test:watch": "npm run test -- --watch",
-    "test:ci": "npm run test -- --coverage",
-    "test:snapshot": "npm run test -- --updateSnapshot",
+    "test": "npm run test:unit && npm run test:storybook",
+    "test:unit": "cross-env NODE_NO_WARNINGS=1 jest --passWithNoTests",
+    "test:unit:watch": "npm run test:unit -- --watch",
+    "test:unit:ci": "npm run test:unit -- --coverage",
+    "test:storybook": "npx concurrently -P -k -s first -n \"SB,TEST\" -c \"magenta,blue\" \"npm run storybook -- --ci\" \"npx wait-on http://localhost:6006 && npx test-storybook {@}\"",
+    "test:storybook:snapshot": "npm run test:storybook -- -- --updateSnapshot",
+    "test:storybook:watch": "npm run test:storybook -- -- --watch",
```

Comment:
There are double `--` here because I need to pass the command two levels deeper, once to the `test:storybook` npm command, and then inside `concurrently`.

Reply:
Interesting. I doubt we can use turbo here for the `test:storybook:**` commands, but could you add a new `turbo` entry (turbo.json) for the `test:unit` and `test:unit:ci` commands?